src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,514 @@
+| InjectedMassPerWeight         | RECOMMENDED. Injected mass per kilogram bodyweight.                                                                                                                                                                                                                                  |
+| InjectedMassPerWeightUnit     | RECOMMENDED. Unit format of the injected mass per kilogram bodyweight (for example, "ug/kg").                                                                                                                                                                                        |
+| SpecificRadioactivityMeasTime | RECOMMENDED. Time to which specific radioactivity measurement above applies in the default unit "hh:mm:ss".                                                                                                                                                                          |
+| MolarActivity                 | RECOMMENDED. Molar activity of compound injected. Corresponds to DICOM Tag (0018,1077) Radiopharmaceutical Specific Activity.                                                                                                                                                        |
+| MolarActivityUnit             | RECOMMENDED. Unit of the specified molar radioactivity (for example, "Bq/g").                                                                                                                                                                                                        |
+| MolarActivityMeasTime         | RECOMMENDED. Time to which molar radioactivity measurement above applies in the default unit "hh:mm:ss".                                                                                                                                                                             |
+| InfusionSpeed                 | RECOMMENDED. If given, infusion speed.                                                                                                                                                                                                                                               |
+| InfusionSpeedUnit             | RECOMMENDED. Unit of infusion speed (for example, "mL/s").                                                                                                                                                                                                                           |
+| InjectedVolume                | RECOMMENDED. Injected volume of the radiotracer.                                                                                                                                                                                                                                     |
```

Comment:
Include a reminder of the recommended units

Reply:
same here.